problems/median_two_sorted_arrays.py

```diff
@@ -0,0 +1,36 @@
+#https://leetcode.com/problems/median-of-two-sorted-arrays/submissions/
+
+from typing import List
+
+class Solution:
+    def findMedianSortedArrays(self, nums1: List[int], nums2: List[int]) -> float:
+        
+        merged = []
+        i = 0
+        j = 0
+        
+        while i < len(nums1) and j < len(nums2):
+            if nums1[i] <= nums2[j]:
+                merged.append(nums1[i])
+                i += 1
+            else:
+                merged.append(nums2[j])
+                j += 1
+                
+        if j == len(nums2):
```

Comment:
you don't need the if statement here and in the opposite case. your while statement should skip if it's all processed